src/composables/modals.ts

```diff
@@ -195,6 +197,18 @@ export function useModals() {
     });
   }
 
+  function openBiometricLoginModal() {
+    return openModal(MODAL_BIOMETRIC_LOGIN);
+  }
+
+  function openPasswordLoginModal() {
+    return openModal(MODAL_PASSWORD_LOGIN);
+  }
+
+  async function openEnableBiometricLoginModal() {
```

Comment:
Why do we need this and `openBiometricLoginModal`?

Reply:
I see that wrong modal name constant was used for the `openEnableBiometricLoginModal` function. Fixed.